student.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Student < Person
+  def initialize(classroom, age, name = 'unknown')
+    super(age, name)
```

Comment:
- [ ] in the `Student` class and `Teacher` class `parent_permission = true` should be passed as an argument as well.